### PR TITLE
EDA of Cor VS GO term and documents

### PR DESCRIPTION
**Pull request by zyhbetty**

### CODE REVIEW

File: R/GeneCorVSgotermEDA.R
Comment (rn1n3r) [inline R/GeneCorVSgotermEDA.R]: You can remove this as the packages should be installed when you list them as Imports in DESCRIPTION.

File: R/GeneCorVSgotermEDA.R
Comment (rn1n3r) [inline R/GeneCorVSgotermEDA.R]: I think you can make the name of your function more expressive (i.e. specify what kind of plots you're making)

File: R/GeneCorVSgotermEDA.R
Comment (rn1n3r) [inline R/GeneCorVSgotermEDA.R]: The name of the returned variable must directly follow `@return` - e.g. `@return myData`

File: R/GeneCorVSgotermEDA.R
Comment (rn1n3r) [inline R/GeneCorVSgotermEDA.R]: Please use fetchData() to load the GEO and HGNC data

File: R/GeneCorVSgotermEDA.R
Comment (rn1n3r) [inline R/GeneCorVSgotermEDA.R]: Consider renaming this variable to something that makes it more clear what it is storing.

**Review by rn1n3r — DISMISSED** [review]
Remember to add yourself as an author to the DESCRIPTION file.

File: R/GeneCorVSgotermEDA.R
Comment (zyhbetty) [reply R/GeneCorVSgotermEDA.R]: fixed to myScatterPlot()

File: R/GeneCorVSgotermEDA.R
Comment (zyhbetty) [reply R/GeneCorVSgotermEDA.R]: fixed to @return (list) and will return list of graphs

File: DESCRIPTION
Comment (hyginn) [inline DESCRIPTION]: **We won't merge that in.** I'll comment on the mailing list. You are not the only one to try this.

**Review by hyginn — CHANGES_REQUESTED** [review]

File: DESCRIPTION
Comment (zyhbetty) [reply DESCRIPTION]: I have added new functions and files that will not merge org.Hs.eg.db

File: R/GeneCorVSgotermEDA.R
Comment (hyginn) [inline R/GeneCorVSgotermEDA.R]: Unneccessary: `BiocManager` is in `Imports:`

File: R/GeneCorVSgotermEDA.R
Comment (hyginn) [inline R/GeneCorVSgotermEDA.R:75]: You don't need this function. It's just one expression, which you can just use in your code.

**Review by hyginn — COMMENTED** [review]
Vignette does not explain anything. Code needs some fixes, but is basically sound; review comments were addressed. Will make two local changes, resolve conflicts and merge.

**Comment by hyginn** [comment]
Waiting for Travis - will merge after check passes.

**Review by hyginn — APPROVED** [review]

**Review by hyginn — COMMENTED** [review]
This took considerable changes to bring into mergeable shape and I suspect it never did pass local check without error, warnings, or notes.

**Comment by hyginn** [comment]
Couldn't get Travis to pass - merged functions manually, required significant changes: probably never passed locally anyway. Vignette not worth merging.  Evaluated as-is. Closed.

**Pull request closed without merging**
